lib/oxidized/nodes.rb

```diff
@@ -58,7 +58,8 @@ def show node
     def fetch node, group
       with_lock do
         i = find_node_index node
-        output = self[i].output.new
+        # Use first output type
+        output = self[i].output[0].new
```

Comment:
> If we do that, then we need to check if defaults is string or array, which seems dubious practice.

i know both travis and [gitlab-ci](https://gitlab.com/gitlab-org/gitlab-ce/blob/1339fda1cd3325c0186b5f1b53444e7319ad3cb6/lib/ci/gitlab_ci_yaml_processor.rb#L103-109) use it.

I like it more to use YAML features :neckbeard:, but since its already used in other places, maybe its the best approach here.